py-polars/polars/dataframe/frame.py

```diff
@@ -4352,7 +4353,10 @@ def _parse_column(col_name: str, dtype: PolarsDataType) -> tuple[str, str, str]:
         return None
 
     def describe(
-        self, percentiles: Sequence[float] | float | None = (0.25, 0.50, 0.75)
+        self,
+        percentiles: Sequence[float] | float | None = (0.25, 0.50, 0.75),
+        *,
+        interpolation: RollingInterpolationMethod = "nearest",
```

Comment:
I don't think we should to expose the interpolation parameter. Feels a bit like offering too much detail when we've never seen a request for this.

Reply:
Well, given that we've already gone to the trouble of offering users custom percentiles it doesn't seem like a leap to optionally allow them to be interpolated to spec (I tend to prefer linear interp, so I'd use it - that's the only reason I thought to add it 😅)

Reply:
Let's ask @ritchie46 for the deciding vote :)

Reply:
Not really a string opinion here. Might be useful for people. I don't think it will hurt us exposing it.